src/V3Control.cpp

```diff
@@ -640,6 +641,45 @@ class V3ControlResolver final {
             return cost;
         }
     }
+    // Helper for adding targets to the hook-insertion config map
+    std::pair<string, string> splitPrefixAndVar(const string& target) {
+        auto pos = target.rfind('.');
+        if (pos == string::npos) {
+            // No prefix, return error
+        }
+        string prefix = target.substr(0, pos);
+        string varTarget = target.substr(pos + 1);
+        return {prefix, varTarget};
+    }
+    // Add the hook-insertion config data to the map to create the initial map (Used in verilog.y)
+    void addHookInsCfg(FileLine* fl, const string& insFunction, int insID, const string& target) {
+        // Error MSG if the hook-insertion of the top module is not possible
+        if ((std::count(target.begin(), target.end(), '.') < 2)) {
+            v3fatal("In .vlt defined target tries to insert-hook to the highest MODULE, which is "
```

Comment:
```suggestion
            fl->v3error("In .vlt defined target tries to insert-hook to the highest MODULE, which is "
```

Don't use v3fatal, need to base on a fileline. Also see other comment about error format